assets/js/state/sagas/sapSystems.js

```diff
@@ -79,6 +81,19 @@ function* applicationInstanceHealthChanged({ payload }) {
   yield put(updateApplicationInstanceHealth(payload));
 }
 
+export function* applicationInstanceAbsentAtChanged({ payload }) {
+  yield put(updateApplicationInstanceAbsentAt(payload));
```

Comment:
This may be a bit technical, but `payload` will include the field `sid`, that we don't strictly need to pass to `updateApplicationInstanceAbsentAt()`. Thoughts on this?

Reply:
It doesn't really matter, it is fine this way